_includes/sidenav.html

```diff
@@ -1,10 +1,14 @@
 {% assign current = page.url | downcase | split: '/' %}
 
+
 <aside class="sidenav" id="menu-content">
   <nav>
     <ul class="usa-sidenav-list">
+      <li> 
+        <a {% if page.url == "/" %}class="usa-current"{% endif %} href="{{ site.baseurl }}/">Introduction</a>
+      </li>
       <li>
-        <a {% if current[1] %}{% else %}class="usa-current"{% endif %} href="{{ site.baseurl }}/">Visual style</a>
+        <a {% if current[1] == 'visual-style' %}class="usa-current"{% endif %} href="{{ site.baseurl }}/visual-style/">Visual style</a>
         <ul class="usa-sidenav-sub_list visual-style-sublist">
           <li>
             <a href="{{ site.baseurl }}/#logo">Logo</a>
```

Comment:
Good catch since it should be, but it actually works as is. I think it works because those sub navigation items are only going to be present when Visual Style has already been selected and because the links have `#` in them the browser will just change the hash in the address bar resulting in expected behavior.